templates/repo/home_sidebar_top.tmpl

```diff
@@ -41,28 +41,30 @@
 					</div>
 				</div>
 			{{end}}
-			{{if .ReadmeExist}}
-				<div class="flex-item-body tw-mt-2">
-					<a class="flex-text-block muted" href="{{.TreeLink}}/{{.FileName}}">
-						{{svg "octicon-book"}} {{ctx.Locale.Tr "readme"}}
-					</a>
-				</div>
-			{{end}}
-			{{if .DetectedRepoLicenses}}
-				<div class="flex-item-body">
-					<a class="flex-text-block muted" href="{{.RepoLink}}/src/{{.Repository.DefaultBranch}}/{{PathEscapeSegments .LicenseFileName}}" title="{{StringUtils.Join .DetectedRepoLicenses ", "}}">
-						{{svg "octicon-law"}} {{if eq (len .DetectedRepoLicenses) 1}}{{index .DetectedRepoLicenses 0}}{{else}}{{ctx.Locale.Tr "repo.multiple_licenses"}}{{end}}
-					</a>
-				</div>
-			{{end}}
-			{{if .CitiationExist}}
-				<div class="flex-item-body">
-					{{template "repo/cite/cite_modal" .}}
-					<a class="flex-text-block muted" id="cite-repo-button">
-						{{svg "octicon-cross-reference"}} {{ctx.Locale.Tr "repo.cite_this_repo"}}
-					</a>
-				</div>
-			{{end}}
+			<div class="tw-flex tw-flex-col tw-gap-1 tw-mt-2">
```

Comment:
I do not think it is right to use other manually written "flex" elements in the "flex-list", it breaks the framework design and will cause more regressions in the future.

Reply:
What do you suggest instead to increase gap between those links?

Reply:
Drop "flex-list" completely if it doesn't fit this use case.

Reply:
Could be done I guess but it will be a more involved refactor.

Reply:
Yes , I also agree with "separate div", but the "flex-body" shouldn't be wrapped by another manually written "flex" element. At least, our "devtest page" doesn't say it is well supported.

Gitea has many legacy UI problems, just because the UI was heavily patched by breaking a lot of framework design (including breaking Fomantic UI).

Reply:
> I'm against putting these links straight into the top-level div. Putting them in a separate div will be much better.

"flex-list" means `<div class="flex-list">`, by framework design